sigmastate/src/test/scala/sigmastate/crypto/SigningSpecification.scala

```diff
@@ -45,6 +47,42 @@ class SigningSpecification extends SigmaTestingCommons {
     printThresholdSignature(msg)
   }
 
+  property("signMessage / verifyMessage roundtrip - simple dlog") {
+    forAll(Gen.alphaNumStr){str =>
+      val msg = str.getBytes("UTF-8")
+      val pi = new ErgoLikeTestProvingInterpreter()
+      val sigmaTree: SigmaBoolean = pi.publicKeys.head
+      val sig = pi.signMessage(sigmaTree, msg, HintsBag.empty).get
+      pi.verifySignature(sigmaTree, msg, sig) shouldBe true
+      pi.verifySignature(sigmaTree, (str + "1").getBytes("UTF-8"), sig) shouldBe false
```

Comment:
Please add negative tests with invalid pk, and sig

Reply:
Done